profiles/target/2.0/datapackage.json

```
@@ -348,7 +348,7 @@
           "homepage": {
             "propertyOrder": 70,
             "title": "Home Page",
-            "description": "The home on the web that is related to this data package.",
+            "description": "The home on the web that is related to this resource.",
```

```suggestion
            "description": "The home on the web that is related to this data package.",
```

@roll If these files are rebuild, it should say `descriptor` (general) or `Data Package` (specific) here.

---

If I remember well this change was due to sharing the definition of `homepage` between data package and schema. Descriptor may be less ambiguous than resource in my opinion, while applying to both.